lib/app/connector.rb

```diff
@@ -10,10 +10,11 @@
 require 'connectors'
 require 'framework'
 require 'utility'
+require 'app/config'
 
 module App
   module Connector
-    POLL_IDLING = 60
+    POLL_IDLING = (App::Config['idle_timeout'] || 60).to_i
```

Comment:
This setting should be reflected in the config file schema.

Reply:
Makes sense, fixing this.